src/modm/board/black_pill/module.lb

```diff
@@ -22,6 +22,32 @@ Cheap and bread-board-friendly board for STM32 F1 series.
 Sold for less than 2 USD on well known Internet shops from China.
 
 http://wiki.stm32duino.com/index.php?title=Black_Pill
+""" + descr_programming
+
+descr_programming = """
+## Programming
+
+Since the board doesn't have a programmer on-board, you need to use your
+own and *specify* which one you're using in a custom `openocd.cfg` file:
+
+```
+# Replace this with your custom programmer
+source [find interface/stlink-v2.cfg]
+
```

Comment:
It would be nice to add the following:
```openocd
# If you want to select specific instance of programmer,
# uncomment the following line:
# hla_serial "..."
# You can get hla_serial by calling `stlink --hla-serial`
```

Reply:
Added this too with example serial